README.md

```diff
@@ -140,6 +148,9 @@ In the above example, we render two variables from a D-HYDRO output map netCDF f
 
 For the `Mesh2d_waterdepth` variable, an additional threshold mesh is rendered at a height of 0.01. Each mesh is assigned its own color, specified by the normalized red, green, blue and alpha (RGBA) values.
 
+## Logging
```

Comment:
it feels like the logging should have been a different issue.

Reply:
I will add such changes in the future in a separate issue or PR -> I needed it for easier debugging